appveyor.yml

```diff
@@ -1,35 +1,57 @@
+# cache cleanup 2018-03-16
```

Comment:
What's this comment referring to? Is it extra?

Reply:
It's a note I left in for my fork testing.  With Appveyor, there is no way of resetting a project's cache from the web ui, so, over time, the cache may contain old gem dependencies, etc.  I can delete the line if you'd like.